pallets/anchors/Cargo.toml

```diff
@@ -26,19 +26,23 @@ sp-runtime = { git = "https://github.com/paritytech/substrate", default-features
 sp-std = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.29" }
 
 # optional dependencies for benchmarking
+cfg-utils = { path = "../../libs/utils", default-features = false }
 frame-benchmarking = { git = "https://github.com/paritytech/substrate", default-features = false, optional = true, branch = "polkadot-v0.9.29" }
+pallet-aura = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.29" }
 
 [dev-dependencies]
 pallet-authorship = { git = "https://github.com/paritytech/substrate", default-features = true, branch = "polkadot-v0.9.29" }
 pallet-balances = { git = "https://github.com/paritytech/substrate", default-features = true, branch = "polkadot-v0.9.29" }
 pallet-randomness-collective-flip = { git = "https://github.com/paritytech/substrate", default-features = true, branch = "polkadot-v0.9.29" }
+sp-consensus-aura = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.29" }
 sp-io = { git = "https://github.com/paritytech/substrate", default-features = true, branch = "polkadot-v0.9.29" }
 
 [features]
 default = ['std']
 runtime-benchmarks = [
   "frame-benchmarking/runtime-benchmarks",
   "pallet-timestamp/runtime-benchmarks",
+  'cfg-utils/runtime-benchmarks',
```

Comment:
I think here should be `pallet-aura` too.

Reply:
`pallet-aura` doesnt have `runtime-benchmarks` defined.